README.md

```diff
@@ -147,3 +147,14 @@ elemeno.getCollectionItem(
   }
 );
 ```
+
+## Caching
+
+The caching module uses the [simple in-memory cache module from ptarjan](https://github.com/ptarjan/node-cache).
+
+To enable it, just call:
+
+#### `elemeno.enableCache([options])`
+
+##### Options
+ - `refreshTime` : Time (in seconds) the cache will delete the stored data (and thus refreshing the data on the next call). Default: 15 mn.
```

Comment:
Sorry if these comments sounds superfluous or nitpicky, but this statement reads a little odd to me:
>Time (in seconds) the cache will **delete** the stored data (and thus **refreshing** the data...

I think `refreshing` should be changed to `refresh` to make it sound a little clearer.